dnf/cli/commands/shell.py

```diff
@@ -271,6 +271,8 @@ def _transaction(self, args=None):
         elif cmd == 'run':
             try:
                 self.base.do_transaction()
+            except dnf.exceptions.Error as e:
+                logger.error(_("Error:") + " " + e.value)
             except:
```

Comment:
The construct bellow is really not a good idea (`except:`). What about to log also other errors?

Reply:
Oh sorry, I forgot the original code there

Reply:
Done.